.github/workflows/push.yml

```
@@ -0,0 +1,13 @@
+name: Pre-commit hooks
+
+on: [push]
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+      - uses: pre-commit/action@v3.0.0
```

As of a few weeks ago, node.js 16 is deprecated, which leads to a warning (see https://github.com/pre-commit/action/issues/189)

```suggestion
      - uses: pre-commit/action@v3.0.1
```